languages/woocommerce-gateway-stripe.pot

```diff
@@ -1,217 +1,498 @@
 # Copyright (C) 2021 WooCommerce
-# This file is distributed under the same license as the WooCommerce Stripe Gateway plugin.
+# This file is distributed under the same license as the WooCommerce Stripe Gateway package.
```

Comment:
Just wanted to make a note that we don't need to update this file at this point. It's updated out-of-the-box in the build/release process.

Reply:
Yep. @reykjalin commented on another PR that it would be nice to have the generated files updated on the repository after changes although not required. But I am ok to revert if necessary

Reply:
Yea, I prefer to have it up-to-date in the repo just so it's less likely that it sneaks into diffs when we're making unrelated changes 🙂